packages/perseus-score/src/widgets/label-image/score-label-image.ts

```diff
@@ -13,7 +13,7 @@ type InteractiveMarkerScore = {
     isCorrect: boolean;
 };
 
-export function scoreMarker(
+export function labelImageScoreMarker(
```

Comment:
Why the rename? Could this function be called `scoreLabelImageMarker` or `scoreMarkerOnLabelImage`? That reads more like English.

Reply:
It's getting exported from `perseus-score` and imported into `perseus` now. Since `scoreMarker` isn't very descriptive and it's exported just for one place in `perseus` I thought I'd give it a more descriptive name.

You're right that I picked the wrong name. I was reading it as a noun (it's not "a score marker"), but it was a verb (it's something that "scores markers").